app/Config/core.php

```diff
@@ -366,11 +366,11 @@
  * object listings, and translation cache files are stored with this configuration.
  */
 Cache::config('_cake_core_', array(
-	'engine' => $engine,
-	'prefix' => $prefix . 'cake_core_',
-	'path' => CACHE . 'persistent' . DS,
-	'serialize' => ($engine === 'File'),
-	'duration' => $duration
+    'engine' => $engine,
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 30 spaces but found 4